Formula/react-native-cli.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class ReactNativeCli < Formula
+  desc "The React Native CLI tools"
+  homepage "https://github.com/facebook/react-native#readme"
```

Comment:
I think https://facebook.github.io/react-native/ is more canonical URL

Reply:
@igas Good call. Since this is specifically just the CLI, would this be better?
https://facebook.github.io/react-native/docs/understanding-cli

I'll update to one or the other.

Reply:
I think the first one is better as it's more like homepage rather than docs